app/main.py

```diff
@@ -1,2 +1,77 @@
+from typing import Hashable, Any
+
+DEFAULT_CAPACITY = 8
+
+
+class Node:
```

Comment:
Hash should be also included here + would make sense to declare this class inside of Dictionary since it is part of it implementation